package.json

```diff
@@ -136,6 +136,7 @@
       "no-console": 1,
       "no-use-before-define": 0,
       "prefer-template": 2,
+      "react/jsx-first-prop-new-line": 0,
```

Comment:
Let's set this to `"multiline"` maybe? That's the style we've adopted, and I'd rather be strict.